xml/yast2_ncurses.xml

```diff
@@ -14,15 +14,15 @@
   </dm:docmanager>
  </info>
  <para>
-  This section is intended for system administrators and experts who do not run
-  an X server on their systems and depend on the text-based installation tool.
-  It provides basic information about starting and operating &yast; in text
-  mode.
- </para>
- <para>
-  &yast; in text mode uses the ncurses library to provide an easy
-  pseudo-graphical user interface. The ncurses library is installed by default.
-  The minimum supported size of the terminal emulator in which to run &yast; is
+  The ncurses-based pseudo-graphical &yast; interface is designed primarily to
+  help system administrators to manage systems without an X
+  server. ncurses-based interface offers several advantages compared to the
```

Comment:
To me it looks strange to begin a sentence with a term spelled in lowercase. Maybe replace 'ncurses-based interface' with 'It'?

Reply:
Good point. I replaced it with _The interface offers..._